master/docs/manual/configuration/steps/source_git.rst

```diff
@@ -152,3 +152,13 @@ The Git step takes the following arguments:
    This may be either a :ref:`Secret` or just a string.
    `sshPrivateKey` must be specified in order to use this option.
    `sshHostKey` must not be specified in order to use this option.
+
+``auth_credentials``
+
+   (optional) An username/password tuple to use when running git for push operations.
```

Comment:
"for push operations" probably does not fit here, because this part is for `Git` step, not `GitPush`.